### PR TITLE
New principle: Where possible APIs should be available in DedicatedWorker

### LINKED ISSUE #325: Guidance about exposing on Window vs Worker only (and other contexts)
This came up in https://github.com/w3ctag/design-reviews/issues/612#issuecomment-862713258 and it was requested that the TAG reaches consensus on guidance on this.

### PR DESCRIPTION
**Pull request by kenchris**
Closes #325

***
<a href="https://pr-preview.s3.amazonaws.com/kenchris/design-principles/pull/356.html" title="Last updated on Jan 27, 2022, 7:34 AM UTC (6f63fe3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/356/51c4da3...kenchris:6f63fe3.html" title="Last updated on Jan 27, 2022, 7:34 AM UTC (6f63fe3)">Diff</a>

### CODE REVIEW

**Review by annevk — COMMENTED** [review]
Can this advice be made more generic to cover all types of workers?

I suspect folks also might want guidance around worklets and `Exposed=*`.

File: index.bs [hunk 1]
```diff
@@ -2536,6 +2536,24 @@ offers guidance that should be considered in the development of new features, no
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3 id="consider-dedicatedworker-support">Where possible APIs should be made available in DedicatedWorker</h3>
+
+When exposing a feature, please consider whether it makes sense to expose the feature
+to DedicatedWorker as well.
+A DedicatedWorker is a non specialized worker type that allows developers to run code
+off the main thread.
```
Comment (cynthia) [inline index.bs:2544]: I think we can just reference back to DedicatedWorker and not explain this ourselves.

Comment (kenchris) [reply index.bs:2544]: But then devs will have to open multiple links, maybe do both?

**Review by cynthia — APPROVED** [review]

**Comment by kenchris** [comment]
@annevk if you have suggestions, please share. I think we want to land something now and then fix in follow ups as we get consensus

File: index.bs (hunk 1 above)
Comment (cynthia) [inline index.bs:2544]:
```suggestion
```

**Comment by annevk** [comment]
I filed #359 as a follow-up. I'm a bit surprised this was just landed without some more time for back-and-forth discussion. At least in the past updates to this document were done quite carefully.

**Pull request merged**